app/main.py

```diff
@@ -1,6 +1,19 @@
+from pprint import pprint
+import requests
+import os
+
+URL = "http://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+
+    querystring = {
+        "key": os.environ["API_KEY"],
```

Comment:
It's better to move API_KEY to a variable and not retrieve it in functions. API_KEY is the thing that should be retrieved at a global level